wear/src/main/java/com/asamm/locus/addon/wear/gui/custom/CustomWearableDrawerLayout.java

```diff
@@ -38,13 +38,13 @@ public CustomWearableDrawerLayout(
 		if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.N_MR1) {
 			// try to make top navigation drawer trigger area bigger
 			try {
-				Field topDragger = getClass().getSuperclass().getDeclaredField("mTopDrawerDragger");
+				Field topDragger = getClass().getSuperclass().getDeclaredField("mBottomDrawerDragger");

```

Comment:
[Suggestion]
As mentioned by previous dev in the comment above: it should worth to check, if size of the `getEdgeSize()` is still a lot smaller then the size you set here. If not, I suggest to try it without this hack. It should be nice to get rid of this class completely.

Reply:
Could this break older devices?

Reply:
I think it depends where this size is defined. But because `WearableDrawerLayout` is part of the AndroidX library, it should be bundled in the final APK so it is part of the add-on, not the system. It is anyway not a critical problem, just a minor suggestion to test this implementation after few years and check if reflection is still needed or not. I do not like this system, because in case, background implementation changes, it break functionality. But it is probably clear to you.

Reply:
Oki, understand. No problem, I'll check it later, thanks.